project-base/storefront/public/locales/cs/common.json

```diff
@@ -190,6 +190,7 @@
     "No": "Ne",
     "No products matched your search": "Hledání neodpovídají žádné produkty",
     "No results match the filter": "Filtru neodpovídají žádné výsledky",
+    "None of the product variants are available for purchase.": "Aktuálně není možné koupit ani jednu variantu produktu.",
```

Comment:
I do not care if you're going to keep the Czech or English version, but the Czech version includes info about this being a temporary state (Aktualne), whereas the English version does not. I would just unify this somehow.